student/views/registration.py

```diff
@@ -79,6 +79,7 @@ class PaymentPreviewView(View):
     def get(self, request, registration_id):
         registration = Registration.objects.get(pk=registration_id)
         original_payable_amount = settings.CODING_BASICS_REGISTRATION_FEE_SGD
+        early_bird_discounted_amount = registration.batch.current_price()
```

Comment:
i think it would be clearer if this `current_price` method was renamed and returned the early bird discount instead. that way, we can keep `original_payable_amount` and more easily see how the calculation is made
i.e. `total_discount = early_bird_discount + coupon_discount`
`final_payable_amount = original_payable_amount - total_discount`

this also allows us to pass in `total_discount` to Stripe and have the discount applied as a line item in the Stripe UI.